source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -40,8 +53,7 @@ To skip a specified number of returned results from a query, pass the
 number of documents you want to skip to the ``SetSkip()`` method of
 the read operation's options.
 
-Specify the options as the last parameter to the following read
-operation methods:
+The following read operations take options as a parameter:
```

Comment:
Issue: Helpful to clarify if options is a list, an object, or something else

Something like:

```suggestion
The following read operations take an options object as a parameter:
```